infra/feast-operator/internal/controller/services/services.go

```diff
@@ -455,23 +471,69 @@ func (feast *FeastServices) createNewPVC(pvcCreate *feastdevv1alpha1.PvcCreate,
 	return pvc, controllerutil.SetControllerReference(feast.Handler.FeatureStore, pvc, feast.Handler.Scheme)
 }
 
-func (feast *FeastServices) getServiceConfigs(feastType FeastServiceType) feastdevv1alpha1.ServiceConfigs {
-	appliedServices := feast.Handler.FeatureStore.Status.Applied.Services
+// <<<<<<< HEAD
```

Comment:
looks like some conflicts

Reply:
I commented this out and forgot to remove it. I'll get it fixed.